mantis-client/src/main/java/io/mantisrx/client/MantisClient.java

```diff
@@ -96,7 +95,7 @@
      */
     public MantisClient(Properties properties) {
         clientWrapper = new MasterClientWrapper(properties);
-        this.disablePingFiltering = Boolean.valueOf(properties.getProperty(ENABLE_PINGS_KEY));
+        this.disablePingFiltering = Boolean.parseBoolean(properties.getProperty(ENABLE_PINGS_KEY));
```

Comment:
never knew the difference between the methods till I saw this. good change!

Reply:
Huh. I also never knew the difference between these two until just now.